HOW_TO_RELEASE.md

```diff
@@ -57,47 +57,26 @@ upstream        https://github.com/pydata/xarray (push)
       pytest
       ```
  9. Check that the ReadTheDocs build is passing.
-10. Tag the release:
-      ```sh
-      git tag -a v{0.X.Y} -m 'v{0.X.Y}'
-      ```
```

Comment:
isn't the tag still necessary?

Also, I would do the update of the release date in this step, in a extra commit with the message `Release v0.X.Y`, and then do the tag on that:
```sh
git tag -a v0.X.Y -m 'v0.X.Y'
```
The advantage is that if the release is delayed but the `whats-new.rst` PR is already merged (e.g. because a bug that has to be fixed before the release was found), the date won't be wrong.

Reply:
I'm not sure - I didn't do that step earlier...

Reply:
you definitely created a tag, but with a different command, probably? The date was updated in the release notes PR.

Reply:
I think when I clicked on the github button to make a new release it must have created the new tag then?

Reply:
that might be it. It might be worth keeping that step manual, though: if the build is broken (e.g. because a build dependency is missing / outdated) you don't have to delete / re-create the release.

Reply:
When you create a draft release GitHub creates the tag and kicks the normal GA.

When you publish the release (after checking that tests have passed) the Github kicks the release GA.

Reply:
ah, good to know, I never used github to create tags. This still feels a bit strange because you'd have to delete / re-create the draft release to rerun the CI (not sure how much of an issue that is), *and* there's the really minor issue of not being able to set the tag message, which means that the message of the last commit will show up in the [overview](https://github.com/pydata/xarray/tags) (hidden behind `...`).